src/fun/counting_v2.go.tmpl

```diff
@@ -0,0 +1,268 @@
+{{/*
+	Count up from 1 using base 10 and/or roman numerals with availability of math
+	View counting statistics with the -cstats command
+	See <https://yagpdb-cc.github.io/fun/counting_v2> for more information.
+	
+	Author: H1nr1 <https://github.com/H1nr1>
+*/}}
+
+{{/* configurable values */}}
+{{ $countTwice := false }} {{/* allow users to count multiple times in a row; true/false */}}
+{{ $correctRID := false }} {{/* correct counting role ID; set to false to disable */}}
+{{ $incorrectRID := false }} {{/* incorrect/blacklist counting role ID; set to false to disable */}}
+{{ $errorCID := .Channel.ID }} {{/* ID of channel to send error reports to; .Channel.ID for current channel */}}
+{{ $saves := 1 }} {{/* how many wrong numbers before reset; set to 0 for no saves */}}
+{{ $reactions := true }} {{/* allow confirmative reactions on message; true/false */}}
+	{{ $reactionDelete := true }} {{/* toggle for reactions to delete from last message; true/false */}}
+	{{ $correctEmoji := "✅" }} {{/* emoji to react with if number is correct; if custom, use format name:id */}}
+	{{ $warningEmoji := "⚠️" }} {{/* emoji to react with if wrong number with available saves; if custom, use format name:id */}}
+	{{ $incorrectEmoji := "❌" }} {{/* emoji to react with if number is incorrect; if custom, use format name:id */}}
+{{ $leaderboardLength := 10 }} {{/* how many members to show on leaderboard; max of 100 */}}
+{{/* end of configurable values */}}
+
+{{$db:=or 
+	(dbGet 0 "counting").Value 
+	(sdict "last" (sdict "user" .BotUser.ID "msg" 0) "next" 1 "highscore" (sdict "user" .BotUser.ID "num" 0 "time" currentTime) "saves" $saves)
+}}
+
+{{with .ExecData}}
+	{{$msg:=getMessage nil .}}
+	{{if not $msg}} {{/* check if message was deleted */}}
+		{{sendMessage nil (cembed 
+			"description" (printf "%s deleted their expression which was correct!\nThe next number is %d" 
+				(userArg $db.last.user).Mention $db.next
```

Comment:
You can save a `userArg` call with `<@%s> deleted ...` and passing in the user ID directly.